lib/datasource/docker/index.js

```diff
@@ -59,6 +57,15 @@ async function getAuthHeaders(registry, repository) {
       );
       opts.headers = { Authorization: `Basic ${auth}` };
     }
+
+    if (authenticateHeader.scheme.toUpperCase() === 'BASIC') {
+      logger.debug(`Using Basic auth for docker registry ${repository}`);
+      await got(apiCheckUrl, opts);
```

Comment:
@derkoe I know this is an old PR, but I was looking at this line today and confused as to what it's achieving. It's calling `got` but not saving the response.

Reply:
OK - I think that line can be removed.